src/common/metadata.cc

```diff
@@ -90,12 +90,17 @@ uint64_t metadataGetVersion(const std::string& file) {
 		throw MetadataCheckException("Can't read the metadata file");
 	}
 
-	if (memcmp(chkbuff, SFSSIGNATURE "M 2.9", strlen(SFSSIGNATURE "M 2.9")) == 0) {
+	std::string signature = std::string(chkbuff, 8);
+	std::string sfsSignature = std::string(SFSSIGNATURE "M 2.9");
+	std::string sauSignature = std::string(SAUSIGNATURE "M 2.9");
+
+	if (signature == sfsSignature || signature == sauSignature) {
```

Comment:
I had already fixed this in the dump metadata [branch](https://github.com/leil-io/saunafs/commit/b84b824637354f3f0330f80691c5a36d2aeb6417) but is not finish yet. LGTM.